packages/zosfiles/__tests__/api/methods/create/Create.unit.test.ts

```diff
@@ -139,6 +140,24 @@ describe("Create data set", () => {
             );
         });
 
+        it("should be able to allocate like from a sequential data set", async () => {
+            const custOptions1 = {
+                dsorg: "PO",
+                alcunit: "CYL",
+                primary: 20,
+                recfm: "FB",
+                blksize: 6160,
+                lrecl: 80,
+                showAttributes: true
+            };
+            const custOptions2 = { like: dataSetName, showAttributes: true };
+
+            await Create.dataSet(dummySession, CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL, dataSetName, custOptions1);
```

Comment:
The first `Create.dataset` call with `custOptions1` is unnecessary here. Since this is a unit test rather than a system test, the second `Create.dataset` call with `custOptions2` will succeed even if the dataset `dataSetName` hasn't been created yet.